docs/concepts/portal-loop.md

```diff
@@ -67,3 +67,20 @@ has some drawbacks:
 Gno will fail to be replayed, meaning **data will be lost**. 
 - Since transactions are archived and replayed during genesis, 
 block height & timestamp cannot be relied upon.
+
+### Deploying to the Portal Loop
+
+There are two kinds of deployements to Portal Loop:
+
+1. *automatic* - all packages in `examples/gno.land/{p,r}/` get added to the
```

Comment:
Let's just mention that examples/ lives in the monorepo; it might not be clear to someone who isn't aware of the magical workings of the monorepo.

```suggestion
1. *automatic* - all packages in found in the `examples/gno.land/{p,r}/` directory in the [Gno monorepo](https://github.com/gnolang/gno) get added to the
```